core/src/com/unciv/logic/automation/city/ConstructionAutomation.kt

```diff
@@ -382,7 +392,21 @@ class ConstructionAutomation(val cityConstructions: CityConstructions) {
         if (foodBuilding != null) {
             var modifier = 1f
             if (city.population.population < 5) modifier = 1.3f
+            modifier *= city.civ.getPersonality().scaledFocus(PersonalityValue.Food)
             addChoice(relativeCostEffectiveness, foodBuilding.name, modifier)
         }
     }
+
+    private fun addFaithBuildingChoice() {
+        val faithBuilding = statBuildings
+            .filter { it.isStatRelated(Stat.Faith) }
+            .filterBuildable()
+            .minByOrNull { it.cost }
+        if (faithBuilding != null && civInfo.gameInfo.isReligionEnabled()) {
```

Comment:
Is enabled should be the first thing we check, before checking buildability
Fast exit